package-lock.json

```diff
@@ -829,6 +829,12 @@
             }
           }
         },
+        "core-js": {
```

Comment:
Do you have an idea why the package.lock was changed? I guess we can revert those right?